src/adapters/openai/helpers/embedding.ts

```diff
@@ -11,15 +11,19 @@ export class Embedding extends SuperOpenAi {
     this.context = context;
   }
 
-  async createEmbedding(text: string): Promise<number[]> {
-    const params: OpenAI.EmbeddingCreateParams = {
-      model: "text-embedding-3-large",
-      input: text,
-      dimensions: VECTOR_SIZE,
-    };
-    const response = await this.client.embeddings.create({
-      ...params,
-    });
-    return response.data[0]?.embedding;
+  async createEmbedding(text: string | null): Promise<number[]> {
+    if (text === null) {
+      return new Array(VECTOR_SIZE).fill(0);
+    } else {
+      const params: OpenAI.EmbeddingCreateParams = {
+        model: "text-embedding-3-large",
```

Comment:
Should this file be deleted or something?

Reply:
I’ve retained it in case it can be expanded to support OpenAI's chat APIs. I believe this could be useful, even if it's not currently in use.

Reply:
We can restore from git history if needed. Its not in use so it should be deleted.